README.md

```diff
@@ -16,6 +16,7 @@ Demo of the code in this repository (with old static data): http://covince.verce
 * `map [object]`
   *  [`settings [object]`](#map-settings) (optional)
   *  [`viewport [object]`](#map-viewport)
+* `nomenclature [object]` (optional) lineages mapped to preferred names. The order of keys will be reflected on the UI.
```

Comment:
Super nit: no need for changes - agree that only dogma is working. Technically I think that JSON dicts are unordered so if we wanted an ordering it should be a list of dicts (or something), but I doubt it will be an issue for any real implementation.

Reply:
I think key order is stable now: https://www.stefanjudis.com/today-i-learned/property-order-is-predictable-in-javascript-objects-since-es2015/, I tested it on a few browsers and the iPhone 8 simulator. Don't mind changing it but then I think we should reintroduce the `useNomenclature` hook to expose the sort and a lookup.

Reply:
Yeah no exactly, it's gonna work. All good. (In _the JSON specification_ the order isn't predictable, but in any real javascript implementation -- and in the javascript standard itself -- it is)

Reply:
Very happy as is.

Reply:
I changed this in the end, the real story with this is because I forgot about the greek alphabet sort requirement and tried to shoehorn sorting into the existing API. I agree that it's ugly.